src/Root.cpp

```diff
@@ -159,6 +159,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	m_MojangAPI = new cMojangAPI;
 	bool ShouldAuthenticate = settingsRepo->GetValueSetB("Authentication", "Authenticate", true);
 	m_MojangAPI->Start(*settingsRepo, ShouldAuthenticate);  // Mojang API needs to be started before plugins, so that plugins may use it for DB upgrades on server init
+	AString MaxLoadedChunks = settingsRepo->GetValueSet("Server", "MaxChunkRAM", "30");
+	m_MinimizeRam = settingsRepo->GetValueSetB("Server", "MinimizeRAM", true);
+	if (!StringToInteger(MaxLoadedChunks, m_MaxRAMChunks))
+	{
+		settingsRepo->SetValue("Server", "MaxChunkRAM", "30");
+		m_MaxRAMChunks = 30;  // in MiB
```

Comment:
>  reasonable default, and let people customize it

That's a very vague default then.

And, with a configuration file that looks like a wall of text and numbers, I wonder how many will *actually, realistically* go about their way to find and change it?

I don't know, but you insist on the changes, so...

> The advantage of it is caching chunks

Yes, I understand. Both of you have made it perfectly clear on multiple occasions what the changes do :P